lib/wiki_course_edits.rb

```diff
@@ -163,6 +182,16 @@ def add_template_to_sandbox
                            sandbox_template:, current_user: @current_user)
   end
 
+  def remove_content_from_page(page_title, content, summary)
+    initial_page_content = @wiki_api.get_page_content(page_title)
+
+    # If content to remove does not exist on initial page, then there is nothing
+    # to remove
+    return unless initial_page_content&.include?(content)
+    new_page_content = initial_page_content.gsub(/#{Regexp.quote(content)}(\n)?/, '')
```

Comment:
I added an optional new line in the regexp, but maybe we can use just something like `#{Regexp.quote(content)}\n`, as the new line should be always added when writing the course template

Reply:
I guess the newline should be removed if present, but if there is not a newline, it should still remove the template.

Reply:
Makes sense.  Then it looks like using `/#{Regexp.quote(content)}(\n)?/` (as it's now) is the right way. If the new line after the content exists, it will match the content and the new line. If the new line doesn't exist, it will match only the content.

Let me know if there is something for me to review. Otherwise, I understand I already addressed all the suggestions/questions.